internal/explorer/db/postgres.go

```diff
@@ -130,10 +131,20 @@ const (
 		COALESCE(public_config.ipv4, ''),
 		COALESCE(public_config.ipv6, ''),
 		COALESCE(node.certification_type, ''),
+		COALESCE(rent.contract_id, 0),
+		COALESCE(rent.twin_id, 0),
 		0
 	FROM node
 	LEFT JOIN node_resources($1) ON node.node_id = node_resources.node_id
 	LEFT JOIN public_config ON node.id = public_config.node_id
+	LEFT JOIN (
```

Comment:
Given that there may be at most one Created contract per node, the query can be simplified by doing the condition on the join like `LEFT JOIN rent_contract ON rent_contract.state = 'Created' AND rent_contract.node_id = node.node_id`.